From: Simon McVittie <smcv@debian.org>
Date: Sat, 22 Jul 2023 10:49:32 +0100
Subject: tests: Skip test_consistency during Debian build

This test assumes that it is run from the source tree, with
pyproject.yml in place, but during the Debian package build the tests
are run in a build directory containing a copy of the source code but
no pyproject.yml.

This test is a consistency check for the upstream maintainers rather
than a functional test, so it doesn't seem crucial for Debian's
purposes.

Forwarded: not-needed
---
 tests/test_consistency.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tests/test_consistency.py b/tests/test_consistency.py
index c68ddc6..a8a879e 100644
--- a/tests/test_consistency.py
+++ b/tests/test_consistency.py
@@ -5,6 +5,8 @@
 import pathlib
 import sys
 
+import pytest
+
 
 if sys.version_info >= (3, 11):
     import tomllib
@@ -12,6 +14,7 @@ else:
     import tomli as tomllib
 
 
+@pytest.mark.skip(reason="Fails when source is copied into a build directory by pybuild")
 def test_pyproject_dependencies():
     pyproject = pathlib.Path(__file__).parent.parent.joinpath('pyproject.toml')
     with open(pyproject, 'rb') as f:
